keyboards/matrix/pjf/ws2812_pwm.c

```diff
@@ -0,0 +1,150 @@
+/*
```

Comment:
Is there origins for this file? It seems quite similar to <https://github.com/omriiluz/WS2812B-LED-Driver-ChibiOS> which is Apache 2.0 (and not a compatible licence).

Reply:
yes, it's came from this file, and porting and tuning for stm32f1, so they can't not be used under QMK?